src/trigger.py

```diff
@@ -21,32 +19,16 @@
 import urllib
 import requests
 
-from logger import Logger
+from base import Service
 
 
-class cmd_run(Command):
-    help = "Submit a new revision to the API based on local git repo"
-    args = [
-        Args.db_config,
-    ]
-    opt_args = [
-        {
-            'name': '--poll-period',
-            'type': int,
-            'help': "Polling period in seconds, disabled when set to 0",
-        },
-        {
-            'name': '--force',
-            'action': 'store_true',
-            'help': "Always create a new checkout node",
-        },
-    ]
+class Trigger(Service):
 
-    def __init__(self, sub_parser, name):
-        super().__init__(sub_parser, name)
-        self._logger = Logger("config/logger.conf", "trigger")
-        self._build_configs = None
-        self._db = None
+    def __init__(self, configs, args):
+        super().__init__(configs, args, 'trigger')
+        self._build_configs = configs['build_configs']
+        self._poll_period = int(args.poll_period)
+        self._force = args.force
```

Comment:
Wouldn't it be better to store these in the context?

Reply:
Yeah probably, it's a bit of a grey area.

Reply:
This was actually done in #172, this PR is not changing this part of the code.  The arguments used to be passed to the run method before the #172 refactoring so I guess it probably should be put back like it was before.  I'll do that as a follow-up PR as it's not really related to this one.

Reply:
Fixed in #176